jdaviz/configs/mosviz/helper.py

```diff
@@ -46,89 +55,133 @@ def _extend_world(self, spec1d, ext):
         world = np.hstack((prepend, world, append))
         return world
 
-    def _update_spec2d_x_axis(self, change):
+    def _update_spec2d_x_axis(self, _):
```

Comment:
I don't understand the usage of `_` here. If it must take argument but you don't use it, why not use `*args`?

Reply:
I used '_' as an ignored argument. But looking at this diff, the argument name should not have changed.